Base/QTGUI/qSlicerScriptedLoadableModuleWidget.h

```diff
@@ -44,8 +44,8 @@ class Q_SLICER_BASE_QTGUI_EXPORT qSlicerScriptedLoadableModuleWidget
   qSlicerScriptedLoadableModuleWidget(QWidget * parentWidget=nullptr);
   ~qSlicerScriptedLoadableModuleWidget() override;
 
-  QString pythonSource()const;
-  bool setPythonSource(const QString& newPythonSource, const QString& className = QLatin1String(""));
+  Q_INVOKABLE QString pythonSource()const;
+  Q_INVOKABLE bool setPythonSource(const QString& newPythonSource, const QString& className = QLatin1String(""));
```

Comment:
Before moving forward, it may be sensible to revisit the naming as suggested by @pieper

Reply:
As a step toward this:
* https://github.com/Slicer/Slicer/pull/7352